frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -2,14 +2,16 @@ import React from "react";
 import { Card, Button, Row, Col } from "react-bootstrap";
 import cowHead from "./../../../assets/CowHead.png"; 
 
-// add parameters 
-const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
+
+// add parameters
+const ManageCows = ({userCommons, commons, onBuy, onSell}) =>  {
     // update cowPrice from fixture
+    // Stryker disable ArithmeticOperator : line 24: not very important and unsure of how to test this
     return (
         <Card>
         <Card.Header as="h5">Manage Cows</Card.Header>
         <Card.Body>
-            {/* change $10 to info from fixture */}
+            {/* why does removing the optional chaining question mark turn the entire page white? */}
```

Comment:
Why does this comment keep showing up?

I've called it out in several code reviews.  Comments like this are fine while you are developing, but they need to be removed before code is ready to merge to main.

And I've answered this question before.

Reply:
This is because it is outdated. I was waiting for BuySellMultipleCows to be approved before merging all the changes into other branches.